pkgs/by-name/py/pywalfox-native/package.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, python3, fetchPypi }:
+
+python3.pkgs.buildPythonApplication rec {
+    pname = "pywalfox-native";
```

Comment:
The indentation is usually two spaces.
Why it wasn't noticed before?

Reply:
@AndersonTorres I apologize for the inconvenience. For my other nix projects I usually setup devshells with alejandra and other lints (deadnix, statix, nil, ...) with direnv linked straight to my emacs, however, this doesn't fit CONTRIBUTING.md so I was checking it manually which is obviously error prone. To be honest I kind of hoped that there would be either pre-commit checks or CI checks for the code styleguide which would pick up any of my possible style mismatches.

Anyway, I apologize once again and should I create a new PR or how should I handle this situation? :pray:

Reply:
We still didn't decide about code style automation. There is a recent RFC about this, tho.

The most we have is editorconfig, that checks for the typical invisible whitespaces in endlines.

For now, just open a new PR. But don't worry too much.

Reply:
Oh okay, thank you. Just a bit disappointed in myself that I tested stuff and there are still issues, but thats fine :smile: I created a simple devshell with a bunch of pre-commit checks that will hopefully catch this stuff in the future.

`pywalfox-init: refactor` is fine? :pray:

Reply:
@AndersonTorres I've created the fixup PR #271681. I was, however, confused at first that ofborg fails with 

`error: cannot coerce a Boolean to a string`

but it seems that this happens to every PR that isnt an init or update? Atleast from what I see in other PRs. The same output happens for `nixpkgs-review pr 271681`. Hopefully this is normal behaviour I guess.

Reply:
This error is happening all around today. Maybe some experimental code broke elsewhere.

Let's wait one day.